tox.ini

```diff
@@ -46,12 +46,6 @@ deps=
 commands=py.test --cov=teuthology --cov-report=term -v {posargs:teuthology/test/integration teuthology/orchestra/test/integration}
 basepython=python2.7
 
-[testenv:flake8]
-install_command = pip install --upgrade {opts} {packages}
-deps=
-  flake8
-commands=flake8 --select=F,E9 {posargs:teuthology scripts}
-
```

Comment:
@vasukulkarni 
I'll try to explain why this is should not be merged yet:
When new feature is suggested to merge to master and we want it to be back ported to py2 it is much easier to do if the code is py2 compatible in first place. In order to make contributor to have code py2/py3 compatible we still have to run both py2 and py3 based.
So I would hold on with getting rid of py2 from master for awhile?
Does this make sense to you?

Reply:
The rest of flake8 related fixes I'm happy to merge.

Reply:
I am going to discuss this specific commit in another PR so that I am not blocked on unit test failure. 
It is just flake8 and not sure if it can bring up the compatibility issue, the bootstrap runs in py3 only and py.test I have seen in tox is still using py2

Reply:
@vasukulkarni 
```
flake8-py3 create: /home/jenkins-build/build/workspace/teuthology-pull-requests/.tox/flake8-py3
[9356] /home/jenkins-build/build/workspace/teuthology-pull-requests/.tox$ /tmp/venv.eacBcHiFr6/bin/python2.7 -m virtualenv --no-download --python /usr/bin/python3 flake8-py3 >flake8-py3/log/flake8-py3-0.log
flake8-py3 installdeps: flake8
```
From this I can assume that tox virtualenv tool is running with py2.7 however the flake8-py3 environment  is already using py3, and correspondingly the flake8 itself is running py3. Thus it is tox only the tox what is running on py2